settings.gradle

```diff
@@ -11,10 +11,13 @@ dependencyResolutionManagement {
 
 // sdk modules
 include ":sdk"
-include ":sdk:core", ":sdk:analytics", ":sdk:push-fcm", ":sdk:fixtures"
+include ":sdk:core", ":sdk:analytics", ":sdk:push-fcm", ":sdk:fixtures", ":sdk:messaging"
 
 refreshVersions {
     rejectVersionIf {
         candidate.stabilityLevel.isLessStableThan(current.stabilityLevel)
     }
 }
+include ':sdk'
+include ':sdk:core', ':sdk:analytics', ':sdk:push-fcm', ':sdk:fixtures'
+include ':sample'
```

Comment:
whats this?

Reply:
A little sneaky leftover from the sample app that was a part of the internal sdk feature branch. I'll get these outta here